src/util/rocksdb/properties.rs

```diff
@@ -364,11 +365,17 @@ impl SizePropertiesCollector {
 
 impl TablePropertiesCollector for SizePropertiesCollector {
     fn add(&mut self, key: &[u8], value: &[u8], entry_type: DBEntryType, _: u64, _: u64) {
-        if entry_type != DBEntryType::Put {
-            return;
-        }
-        let size = key.len() + value.len();
-        self.index_handle.size += size as u64;
+        let value_size = match entry_type {
+            DBEntryType::Put => value.len() as u64,
+            DBEntryType::BlobIndex => match TitanBlobIndex::decode_from(value) {
+                Ok(index) => index.blob_size,
```

Comment:
I think also need to add value.len() even if the type is blob, because value do occupy space.

Reply:
Currently it is about 16 bytes, in the future we may store more informations in it, for example timestamp.